pkg/rancher-desktop/backend/backend.ts

```diff
@@ -74,7 +74,7 @@ export interface BackendEvents {
 /**
  * Settings that KubernetesBackend can access.
  */
-export type BackendSettings = RecursiveReadonly<Settings>;
+export type BackendSettings = Settings;
```

Comment:
Don't we want to ensure that we don't accidentally modify the passed-in settings? The previous design was ensuring that the backends had an immutable _snapshot_ of settings, and new ones only get applied via explicit calls to `start()` etc.

Reply:
I put the types back in